### DISCUSSION TITLE
Air Application crashes with faulting module VirtualCamDevice.dll on Intel 11 Gen Processor

### DISCUSSION THREAD

**Discussion by Khumee** [airsdk/Adobe-Runtime-Support#3180] [Q&A]
Our Air application get crashed on a specific laptop, from event viewer it seems something wrong with virtual cam drivers . 

 
![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/18255710/4ec2054e-d274-4f45-a7e1-f266b8d78cc1)
https://www.loom.com/share/cc7ce86d04a1476b9e33defca0ea6cb7

Can anyone help in this regard ?

**Comment by KaminoRyo** [comment]
We have received inquiries from customers saying that "AIR applications do not start on some Windows 11 PCs", and this may be this case...

**Reply by itlancer** [reply]
I have similar issue with NVIDIA Broadcast https://github.com/airsdk/Adobe-Runtime-Support/issues/2018
Also related issue: https://github.com/airsdk/Adobe-Runtime-Support/issues/2941

**Reply by Khumee** [reply]
After uninstalling Intel Arc Control  https://www.intel.com/content/www/us/en/products/docs/discrete-gpus/arc/software/arc-control.html from the system, it starts working.
